python/py-pendulum/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pendulum
+version             2.1.2
+revision            0
+license             MIT
+platforms           darwin
+supported_archs     x86_64
+maintainers         {@xeron gmail.com:xeron.oskom} openmaintainer
+description         Python datetimes made easy
+long_description    ${description}
+
+python.versions     36 37 38 39
+
+homepage            https://pypi.python.org/pypi/${python.rootname}/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
```

Comment:
you can remove `master_sites` and `distname` here; these are the defaults set by the python PG